package.json

```diff
@@ -78,6 +78,7 @@
     "react-native-config": "^1.4.0",
     "react-native-document-picker": "^5.1.0",
     "react-native-gesture-handler": "1.9.0",
+    "react-native-get-random-values": "^1.7.0",
```

Comment:
@nickmurray47 I have decided to use [this polyfill](https://www.npmjs.com/package/react-native-get-random-values) for `getRandomValues`, what I have found it is very popular solution for this. We will need to test thoroughly it works well on all platforms, but it should.

Reply:
Nice! This looks like a good, well-updated package.